packages/server/database/migrations/20221103134908-adjustUserTiers.ts

```diff
@@ -0,0 +1,101 @@
+import {R, RDatum, RValue} from 'rethinkdb-ts'
+import getPg, {closePg} from '../../postgres/getPg'
+import IUser from '../../postgres/types/IUser'
+import segmentIo from '../../utils/segmentIo'
+import {TierEnum} from '../types/Invoice'
+import OrganizationUser from '../types/OrganizationUser'
+import User from '../types/User'
+
+export const up = async function (r: R) {
+  const removedOrgUserIds = await r
+    .table('OrganizationUser')
```

Comment:
I'm not sure why the typechecker is hanging 🤔 I get the following error: `Migration error Error: Cannot find module './generated/getUsersByIdsQuery'`, but I'm not importing `getUsersByIdsQuery`. I was importing it in the previous commit. I tried re-running it but no dice

Reply:
I encountered this issue recently too, fixed it by running `yarn pg:build` before `yarn dev`

Reply:
Good to know! But the error for me is happening in GitHub Actions, rather than locally, where the typecheck is working successfully. Running `yarn pg:build` before `yarn dev` doesn't update any files for me, so I don't think it'll fix the Actions typechecker issue. 

@Dschoordsch or @rafaelromcar-parabol do you have any ideas as to why the Actions typechecker would [continue endlessly and show an error when I terminate it](https://github.com/ParabolInc/parabol/actions/runs/3394832362/jobs/5645174770) but work locally? I've tried rerunning it once